src/client/pages/Offer/Checkout/index.tsx

```diff
@@ -429,13 +441,18 @@ export const Checkout = ({
         setSignUiState('MANUAL_REVIEW')
         return
       }
-
+      trackOfferEvent({
+        eventName: EventName.SignError,
+      })
       setSignUiState('FAILED')
       return
     }
   }
 
   const handleSignStart = () => {
+    trackOfferEvent({
+      eventName: EventName.ButtonClick,
```

Comment:
shouldn't we add?

```
{
    options: { buttonId: 'SOME_BUTTON_ID' },
}
```

Reply:
hmm, we could do. but we only have on button in offer sidebar 🤔

Reply:
hehe well in the tracking tool it will just say "button click" no? 

So maybe `buttonId` could just be `sidebar_cta` or similar?